src/style.css

```diff
@@ -1 +1,67 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.page {
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__search {
+  box-sizing: border-box;
+  outline: none;
+  width: 100%;
+  height: 100%;
+
+  background-repeat: no-repeat;
+  background-image: url(./images/Search.svg);
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  transition: box-shadow 0.3s ease;
+
+  font-family: Avenir, sans-serif;
```

Comment:
You should add fallback fonts for your `font-family` property. This ensures that if the main font isn't available, the browser can use an alternative one. In this case, you've already added `sans-serif` as a fallback, but it would be better to specify a more similar fallback font to Avenir, such as Helvetica or Arial.